src/app/detail/ui/styledComponent.tsx

```diff
@@ -87,6 +87,10 @@ export const CommentCardContainer = styled(Stack)(({ theme }) => ({
   background: theme.color.base.white,
   padding: '8px',
   width: '100%',
+  //remove gap from the parent div of the Tapwrite component
+  '& > div:first-of-type': {
+    gap: '0px !important',
+  },
```

Comment:
Can we not update tapwrite to accept gap props, same as padding and width? or better yet style props?

Reply:
Added a PR in tapwrite repo - https://github.com/aatbip/tapwrite/pull/16

Reply:
I guess you need to add new version of tapwrite then, don't you?